Platforms/QemuSbsaPkg/VirtNorFlashStandaloneMm/VirtNorFlashStandaloneMm.inf

```diff
@@ -0,0 +1,71 @@
+#/** @file
+#
+#  Component description file for NorFlashDxe module
+#
+#  Copyright (c) 2011 - 2021, Arm Limited. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#**/
+
+[Defines]
+  INF_VERSION                    = 1.29
+  BASE_NAME                      = VirtNorFlashStandaloneMm
+  FILE_GUID                      = a0dedcb8-c1ae-4431-b458-80b35fc5042d
+  MODULE_TYPE                    = MM_STANDALONE
+  PI_SPECIFICATION_VERSION       = 0x00010032
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = NorFlashInitialise
+
+[Sources.common]
+  VirtNorFlash.c
+  VirtNorFlash.h
+  VirtNorFlashFvb.c
+  VirtNorFlashStandaloneMm.c
+
+[Packages]
+  EmbeddedPkg/EmbeddedPkg.dec
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  #OvmfPkg/OvmfPkg.dec
+
+[LibraryClasses]
+  BaseLib
+  DebugLib
+  #DxeServicesTableLib

```

Comment:
Can the commented out library classes just be deleted? I'm assuming this INF was copied/pasted from a DXE/Traditional MM module, but these won't apply to Standalone MM and I don't see much value in them being commented without additional context.

Reply:
The commented out stuff will eventually be removed.  I'm leaving them commented out temporarily to make referencing the initial functionality easier.  When things are working correctly I'll purge them.